app/views/play/CampaignView.coffee

```diff
@@ -939,11 +939,11 @@ module.exports = class CampaignView extends RootView
     courseID = $(e.target).parents('.course-version').data 'course-id'
     courseInstanceID = $(e.target).parents('.course-version').data 'course-instance-id'
 
-    classroomLevel = @classroomLevelMap[levelOriginal]
+    classroomLevel = @classroomLevelMap?[levelOriginal]
```

Comment:
`@classroomLevelMap` is not defined in home version, so checking this here. Same for other changes in CampaignView

Reply:
This seems like fine defensive coding.  Is it possible to get here in home version?

Reply:
Not here, but it does go to line 957 for home version. So I added it here as well, in case `@classroomLevelMap` is not defined due to any reason.